src/hdmf/data_utils.py

```diff
@@ -179,9 +179,15 @@ class GenericDataChunkIterator(AbstractDataChunkIterator):
             doc="Display a progress bar with iteration rate and estimated completion time.",
             default=False,
         ),
+        dict(
+            name="progress_bar_class",
+            type=None,  # Would prefer callable, but docval can't handle
```

Comment:
I think this should work:
```suggestion
            type=Callable,
```

with
```python
from collections.abc import Callable
```

Test:
```python
from collections.abc import Callable
from hdmf.utils import docval
@docval(dict(name="a", type=Callable, doc="a"), is_method=False)
def func(**kwargs):
    print(kwargs)
def func2():
    print("hi")
func(func2)  # returns {'a': <function func2 at 0x103716e80>}
```

Reply:
Thanks @rly